### PR TITLE
CERD-188 > MCQ Documentation

### PR DESCRIPTION
**Pull request by adjamenhmh**

### CODE REVIEW

File: packages/multiple-choice/package.json [hunk 1]
```diff
@@ -10,13 +10,15 @@
         "prepare": "npm run build; npm run sass",
         "sass": "node-sass src/sass -r -o dist/css",
         "sass-watch": "node-sass src/sass -r -o dist/css -w",
-        "watch": "npm run prepare; concurrently --names 'build,sass' -c 'bgBlue,bgMagenta' 'npm run build -- -w' 'npm run sass -- -w'",
-        "test": "lit-element-tester dist/components/multiple-choice.js"
+        "watch": "npm run prepare; concurrently --names 'build,sass' -c 'bgBlue,bgMagenta' 'npm run build -- -w' 'npm run sass -- -w' 'npm-watch'",
+        "test": "lit-element-tester dist/components/multiple-choice.js",
+        "_generateDocs": "polymer analyze ./dist/components/ > ./dist/components/analysis.json"
```
Comment (drochgenius) [inline packages/multiple-choice/package.json]: why the leading unserscore?
can you explain the `watch` prop below, I'm not familiar. Hopefully I learn something new ;-)

File: packages/multiple-choice/package.json [hunk 2]
```diff
@@ -28,7 +30,15 @@
...
         "source-map-support": "^0.5.8",
-        "typescript": "latest"
+        "typescript": "latest",
+        "polymer-cli": "latest",
+        "@webcomponents/webcomponentsjs": "^2.0.2",
+        "@polymer/iron-component-page": "next",
+        "@polymer/iron-demo-helpers": "latest"
+    },
+    "watch": {
+        "_generateDocs": "src/components/**/*.*"
```
Comment (drochgenius) [inline packages/multiple-choice/package.json]: never seen this before, thansk for explaining

**Comment by drochgenius** [comment]
THansk @adjamenhmh  for tagging your PR.
@chrisunl , please make sure you tag your PR with appropriate labels.
When we have 100+ PRs merged, it will make our life easier to search for something.

File: packages/multiple-choice/package.json (hunk 1 above)
Comment (adjamenhmh) [reply packages/multiple-choice/package.json]: @drochgenius the leading underscore

Comment (adjamenhmh) [reply packages/multiple-choice/package.json]: @drochgenius all I added to the watch was npm-start, which will watch changes in the folder indicated in 
```
        "_generateDocs": "src/components/**/*.*"
```
and will run the command 
```
polymer analyze ./dist/components/ > ./dist/components/analysis.json
```  
which is responsible for generating the documentation

more info on npm-watch: https://github.com/M-Zuber/npm-watch

Comment (adjamenhmh) [reply packages/multiple-choice/package.json]: the underscore is a convention @Dom-HMH-MTL  recommended to differentiate implicit commands from explicit ones, I believe

**Pull request merged**